src/lib/utils/availability-helpers.ts

```
@@ -0,0 +1,338 @@
+            tableName,
+            startDate,
+            startMinutes,
+            endMinutes,
+            addon.quantity,
+            addonData.total_quantity,
+            operation
+          );
+        }
+      }
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)